### PR TITLE
Source google ads: fix for page token expired

### PR DESCRIPTION
**Pull request by augan-rymkhan**
## What
Resolves **page token expired** issue provided in On-Call issue [#103](https://github.com/airbytehq/oncall/issues/103)

Stream slice's date range is 1 month, if it takes more than 2 hour to process all the records from the given slice, next page tokens which records are being retrieved will be expired.
_For example_: 
Slice start: 01.01.2022
                      Slice end: 30.01.2022
`page_size` = 1000
Within 2 hours it has read 2 000 000 records (2000 pages).
After that time it tries to read page number 2001, but page tokens are expired already.

**The solution**: reduce the slice's date range

image

The similar issue was discussed in the following links:

- https://groups.google.com/g/adwords-api/c/f4fTkTo8m_o/m/QGlUwuo-BAAJ
- https://github.com/googleads/google-ads-php/issues/366

## How
Reduce slice range up to 15 days.
Date range is set to 15 days for each slice, because for `conversion_window_days` default value is 14. Range less than 15 days will break the integration tests.

## Recommended reading order
1. `source_google_ads/streams.py`
2. `unit_tests/test_google_ads.py`
3. `unit_tests/test_source.py `

### CODE REVIEW

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py [hunk 1]
```diff
@@ -64,7 +69,7 @@ class IncrementalGoogleAdsStream(GoogleAdsStream, ABC):
     days_of_data_storage = None
     cursor_field = "segments.date"
     primary_key = None
-    time_unit = "months"
+    range_days = 15
```
Comment (sergei-solonitcyn) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py:72]: I suggest to add a comment to describe shortly the reason of this exact value.

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py [hunk 2]
```diff
@@ -214,5 +217,5 @@ class ClickView(IncrementalGoogleAdsStream):
     ClickView stream: https://developers.google.com/google-ads/api/reference/rpc/v8/ClickView
     """
 
-    time_unit = "days"
     days_of_data_storage = 90
+    range_days = 1
```
Comment (sergei-solonitcyn) [inline airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py:221]: And description of this overriding of the value coul be usefel as well.

**Review by sergei-solonitcyn — CHANGES_REQUESTED** [review]

File: airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py (hunk 2 above)
Comment (augan-rymkhan) [reply airbyte-integrations/connectors/source-google-ads/source_google_ads/streams.py:221]: Before the changes provided in this PR, `time_unit` was overriden with "days" and 
`start_date = start_date.add(**{time_unit: 1})`, the final result means `start_date.add(days=1)`
So, to save the behaviour, just set `range_days` to 1,

**Comment by sherifnada** [comment]
@augan-rymkhan can you also include the root cause in the PR description or in a source code comment (even better) to help retain this knowledge?

**Review by sergei-solonitcyn — APPROVED** [review]

**Pull request merged**